client/app/vps/cloud-database/order/vps-cloud-database-order.html

```diff
@@ -0,0 +1,129 @@
+<section class="cui-page__content">
+    <oui-back-button
+        data-heading="{{:: 'vps_tab_cloud_database_order' | translate}}"
+        data-ui-sref="iaas.vps.detail.cloud-database">
+    </oui-back-button>
+    <div class="mb-3" data-translate="vps_tab_cloud_database_order_disclaimer"></div>
+    <div class="mb-5">
+        <p class="bold" data-translate="vps_tab_cloud_database_order_explanation_title"></p>
+        <p data-translate="vps_tab_cloud_database_order_explanation_text"></p>
+    </div>
+    <h3 data-translate="vps_tab_cloud_database_order_customize"></h3>
+    <hr />
+    <form name="cloudDatabaseOrderForm">
+
+        <div>
+            <label class="required"
+                   for="orderVersion"
+                   data-translate="vps_tab_cloud_database_order_version"></label>
+            <oui-select
+                data-placeholder="{{:: 'common_select' | translate }}"
+                data-model="$ctrl.currentOrder.version"
+                data-items="$ctrl.versions"
+                data-match="name"
+                data-on-change="$ctrl.showOrRefreshDurations()"
+                data-required>
```

Comment:
`required` is a native HTML5 property, no need for `data-`

Reply:
https://master.ui-kit.ovh/#!/oui-angular/select

Reply:
By our conventions, HTML5 native properties are not prefixed by `data-`

Reply:
What I try to tell you is that the oui-select component overrides "required" with its own "data-required" version. So that when you put "data-required" on the component, it is forwarded to the actual native HTML element.